travis.sh

```diff
@@ -0,0 +1,38 @@
+#/bin/bash
+
+set -e
+
+# Travis CI clones the repository at ~/vmware-samples/go-redis-pmem
+# Move the repository into GOPATH first.
+cd ..
+mkdir -p $GOPATH/src/github.com/vmware-samples
+mv go-redis-pmem $GOPATH/src/github.com/vmware-samples
+
+# Build the modified Go compiler.
+git clone --depth 1  https://github.com/jerrinsg/go-pmem ~/go-pmem
```

Comment:
Don't you want to move any old ~/go-pmem away before cloning the latest one on top of it?

Reply:
Hi Pratap,
Each time travis CI runs a build, it creates a new (clean) virtual environment to do the build (ref - https://docs.travis-ci.com/user/for-beginners/#ci-builds-and-automation-building-testing-deploying). So there wouldn't be any user created files in the home directory.

Also, `script.sh` does a `set -e` initially so the script exits with a `-1` error code if any line fails to execute.